ako-gateway-api/nodes/avi_model_l7_translator.go

```diff
@@ -35,11 +35,12 @@ import (
 func (o *AviObjectGraph) AddDefaultHTTPPolicySet(key string) {
 	parentVS := o.GetAviEvhVS()[0]
 
+	policyRefName := lib.GetClusterName() + "-" + utils.CloudName + "-" + akogatewayapilib.DefaultPSName
```

Comment:
@ashishnayak96: We can go with `clusterName` only as clustername is unique identifier . We can go with `lib.Clustername--DefaultPSName` way of naming convention. Please define it as part of lib so it will be reusable . We have defined all naming convention over there. It will be easier to update.